mobile-app/app/components/EstimatedFeeInfo.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import { tailwind } from '@tailwind'
+import { ThemedIcon, ThemedText, ThemedView } from './themed'
+import { BottomSheetModal } from './BottomSheetModal'
+import { translate } from '@translations'
+import { View } from 'react-native'
+import NumberFormat from 'react-number-format'
+
+interface EstimatedFeeElement {
+  value: string | number
+  suffix?: string
+  testID: string
+}
+
+export function EstimatedFeeInfo ({
+  lhs,
+  rhs
+}: { lhs: string, rhs: EstimatedFeeElement }): JSX.Element {
+  return (
+    <ThemedView
+      dark={tailwind('bg-gray-800 border-b border-gray-700')}
+      light={tailwind('bg-white border-b border-gray-200')}
+      style={tailwind('p-4 flex-row items-start w-full')}
+    >
+      <View style={tailwind('w-5/12')}>
+        <View style={tailwind('flex-row items-center justify-start')}>
+          <ThemedText style={tailwind('text-sm mr-1')} testID={`${rhs.testID}_label`}>
+            {lhs}
+          </ThemedText>
+
+          <BottomSheetModal
+            name='EstimatedFeeInfo'
+            snapPoints={['50%', '50%']}
```

Comment:
I know we won't know the content height in exact, but 50% is covering half of the screen, can we go with somewhere around 30% like the figma's?